examples/pytorch/quickstart_advanced.py

```diff
@@ -48,6 +48,10 @@ def add_llm_args(parser):
                             'VANILLA', 'TRTLLM', 'FLASHINFER',
                             'FLASHINFER_STAR_ATTENTION'
                         ])
+    parser.add_argument('--moe_backend',
+                        type=str,
+                        default='CUTLASS',
```

Comment:
In `_torch/model_config.py`, the default value `moe_backend` is `TRTLLM`. Which one is recommended?

Reply:
We should default to `CUTLASS` everywhere because it has a much smaller supporting matrix. Let me fix it.